documentation/content/en/articles/new-users/_index.adoc

```diff
@@ -123,7 +123,7 @@ You can quit `adduser` any time by typing kbd:[Ctrl+C], and at the end you will
 You might want to create a second new user so that when you edit `jack`'s login files, you will have a hot spare in case something goes wrong.
 
 Once you have done this, use `exit` to get back to a login prompt and log in as `jack`.
-In general, it is a good idea to do as much work as possible as an ordinary user who does not have the power-and risk-of `root`.
```

Comment:
Each _single_ hyphen '`-`' flanked by words – **no space on either side** of the hyphen – was incorrect. 

Looking ahead: the commit from finalisation of this PR can be a fix to a past commit. 

(A lazy guess: something around the time of migration to AsciiDoc.)